### PR TITLE
feat: Update charges layer and add docs

### LINKED ISSUE #5064: Add docs for Charges layer
**Is your feature request related to a problem? Please describe.**

The API docs need to have samples for charges layer. 

**Additional context**

Parent issue #4852

### PR DESCRIPTION
**Pull request by srv-twry**
Fixes #5064 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [ ] The unit tests pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Short description of what this resolves:
Update charges layer and add docs for it in the blueprint.

#### Changes proposed in this pull request:
Made required changes.

success
failure

Apiary link: https://openeventserver.docs.apiary.io/

### CODE REVIEW

File: tests/hook_main.py [hunk 1]
```diff
@@ -3826,6 +3826,16 @@ def delete_order(transaction):
...
 
 
+@hooks.before("Orders > Charge > Charge for an Order")
+def orders_charge(transaction):
+    """
+    GET /orders/1/charge
+    :param transaction:
+    :return:
+    """
+    transaction['skip'] = True
```
Comment (schedutron) [inline tests/hook_main.py]: Why skipping the transaction?

Comment (srv-twry) [reply tests/hook_main.py]: How will i get the actual stripe token for testing ?

**Comment by nikit19** [comment]
@srv-twry The clients have to pass the id of the Stripe token here or what?
Could you confirm,the token that a client receives is an object https://stripe.com/docs/api#tokens

**Comment by srv-twry** [comment]
@nikit19 Pass the `id` of the token.

**Pull request merged**